.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.5
```

Comment:
The commit message says `2.5.5` but this says `2.6.5`

Reply:
good catch, but changed back to 2.5.5 anyway, because 2.6.5 still wasn't working on CircleCI